.github/workflows/merlin.yml

```diff
@@ -141,6 +141,7 @@ jobs:
           name: sdk-test-${{ matrix.python-version }}
           token: ${{ secrets.CODECOV_TOKEN }}
           working-directory: ./python/sdk
+          codecov_yml_path: ../../.github/workflows/codecov-config/codecov.yml
```

Comment:
I've added the codecov threshold setting here @ariefrahmansyah. Thanks for the digging this! 

The configs are inspired from this PR here: https://github.com/kensho-technologies/graphql-compiler/pull/806/files; I removed the other configs because `target`'s default is already `auto` and `base` is supposedly deprecated according to the [docs](https://docs.codecov.com/docs/commit-status#threshold). Note that the paths are slightly different because of the different `working-directory`-s.